src/devices/sensible_living.c

```diff
@@ -0,0 +1,183 @@
+    nb_bytes=0;
+    pos += init_pattern_len;
+    for(; pos < len && nb_bytes < msg_len; pos += 12){
+        bitbuffer_extract_bytes(bitbuffer, row, pos, rxBits, /*len=*/16);
+        // ^ we should read 16 bits and not 12, elsewhere last 4bits are ignored
+        rxBits[0] = reverse8(rxBits[0]);
+        rxBits[1] = reverse8(rxBits[1]);
+        rxBits[1] = ((rxBits[1] & 0x0F)<<2) + (rxBits[0]>>6);
+        rxBits[0] &= 0x3F;
+        uint8_t hi_nibble = symbol_6to4(rxBits[0]);
+        if(hi_nibble > 0xF){
+            if(debug_output){
+                fprintf(stdout, "Error on 6to4 decoding high nibble: %X\n", rxBits[0]);
+            }
+            return 0;
+        }
+        uint8_t lo_nibble = symbol_6to4(rxBits[1]);
+        if(lo_nibble > 0xF){
+            if(debug_output){
+                fprintf(stdout, "Error on 6to4 decoding low nibble: %X\n", rxBits[1]);
+            }
+            return 0;
+        }
+        uint8_t byte =  hi_nibble<<4 | lo_nibble;
+        payload[nb_bytes] = byte;
+        if(nb_bytes == 0){
+            msg_len = byte;
+        }
+        nb_bytes++;
+    }
+
+    // Get header
+    data_len = msg_len - SL_ASK_HEADER_LEN - 3;
+    house_id = payload[1];
+    module_id = payload[2] * 256 + payload[3];
+    sensor_type = payload[4];
+    sensor_count = payload[5];
+    alarms = payload[6];
+    sensor_value = payload[7] * 256 + payload[8];
+    battery_voltage = payload[9] * 256 + payload[10];
+
+    // Check CRC
+    crc = payload[5 + data_len] + (payload[5 + data_len + 1]<<8);
+    crc_recompute = ~crc16(payload, msg_len-2, 0x8408, 0xFFFF);
+    if(crc_recompute != crc){
+        if(debug_output){
+            fprintf(stdout, "CRC error: %04X != %04X\n", crc_recompute, crc);
+        }
+        return 0;
+    }
+
+    data = data_make(
+        "time",             "",                 DATA_STRING,  time_str,
+        "model",            "",                 DATA_STRING,  "Sensible Living Mini-Plant Moisture Sensor",
+        "house_id",         "House ID",         DATA_INT,     house_id,
+        "module_id",        "Module ID",        DATA_INT,     module_id,
+        "sensor_type",      "Sensor Type",      DATA_INT,     sensor_type,
+        "sensor_count",     "Sensor Count",     DATA_INT,     sensor_count,
+        "alarms",           "Alarms",           DATA_INT,     alarms,
+        "sensor_value",     "Sensor Value",     DATA_INT,     sensor_value,
+        "battery_voltage",  "Battery Voltage",  DATA_INT,     battery_voltage,
+        NULL
+	);
+    data_acquired_handler(data);
+
+    return 0;
+}
+
+static char *output_fields[] = {
+    "time",
+    "model",
+    "house_id",
+    "module_id",
+    "sensor_type",
+    "sensor_count",
+    "alarms",
+    "sensor_value",
+    "battery_voltage",
+    NULL
+};
+
+r_device sensible_living = {
+    .name           = "Sensible Living Mini-Plant Moisture Sensor",
+    .modulation     = OOK_PULSE_PCM_RZ,
+    .short_limit    = SL_ASK_BIT_LEN,
```

Comment:
Please use absolute values. It easier to graps when browsing through the source.